MonoGame.Framework/Platform/Native/Graphics.Interop.cs

```diff
@@ -120,14 +121,15 @@ internal struct MGG_SamplerState_Info
 }
 
 [StructLayout(LayoutKind.Sequential)]
-internal struct MGG_InputElement
+internal unsafe struct MGG_InputElement
 {
     public VertexElementUsage SemanticUsage;
     public uint SemanticIndex;
     public uint VertexBufferSlot;
     public VertexElementFormat Format;
     public uint AlignedByteOffset;
     public uint InstanceDataStepRate;
+    public fixed byte Name[32];
```

Comment:
@dellis1972 if you don't need this in this current PR i recommend removing it.

I think what you need for GL can be derived from the existing input elements and or the shader data.

Making this structure unsafe and the function call using it unsafe and the usages of that function unsafe just for a member we possibly don't need in the end and don't need at all right now...  seems like it doesn't belong in this PR.